src/masternode/sync.cpp

```diff
@@ -16,10 +16,11 @@
 class CMasternodeSync;
 std::unique_ptr<CMasternodeSync> masternodeSync;
 
-CMasternodeSync::CMasternodeSync(CConnman& _connman) :
```

Comment:
Consider naming it `govman`, similar to [`dsnotificationinterface.cpp`](https://github.com/dashpay/dash/blob/b20ff5406cf5a6673babed07578c3e4b542d1456/src/dsnotificationinterface.h#L45). 

It's very unlikely `govman` would be confused with something else as no manager shares its name or has a similar one.